### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by bencbernstein**
slot machine on smaller screens

### CODE REVIEW

File: src/Components/Home/index.js [hunk 1]
```diff
@@ -26,10 +26,21 @@ class Home extends Component {
       displayMobilePopup: false,
       redirect: null,
       isMobile: isMobile,
+      isSmallScreen: false,
       iosIdx: 0
     };
   }
 
+  checkWindowSize() {
+    const isSmallScreen = window.innerWidth <= 1100 ? true : false
```
Comment (OLI9292) [inline src/Components/Home/index.js]: this can just be `const isSmallScreen = window.innerWidth <= 1100`

File: src/Components/Home/index.js [hunk 2]
```diff
@@ -26,10 +26,21 @@ class Home extends Component {
...
+    this.setState ({ isSmallScreen: isSmallScreen });
```
Comment (OLI9292) [inline src/Components/Home/index.js]: if the var name is the same you can do `this.setState ({ isSmallScreen })`

Comment (OLI9292) [reply src/Components/Home/index.js]: also that extra space is worth removing

File: src/Components/Home/index.js [hunk 3]
```diff
@@ -26,10 +26,21 @@ class Home extends Component {
...
+  }
+
+  componentDidMount() {
+    this.checkWindowSize();
+    window.addEventListener("resize", this.checkWindowSize.bind(this));
```
Comment (OLI9292) [inline src/Components/Home/index.js]: this event listener should be removed when the component dismounts, also let's stick to using single quotes w/ any JS (ie. `'` over `"`)

**Comment by OLI9292** [comment]
looks good. just a few comments.  might be worth putting the data in a separate file.  
also, lets build before pushing to heroku not master.  so if you can remove the `/build` files from this pr

**Comment by OLI9292** [comment]
seeing build files like `asset-manifest.json` and `service-worker.js` which is preventing me from merging

**Comment by OLI9292** [comment]
we could gitignore them potentially, i'm not sure whether heroku is gitignore though

File: src/Components/Home/index.js [hunk 4]
```diff
@@ -348,6 +359,14 @@ const TopLeftContainer = styled.div`
     height: fit-content;
   }
 `
+<<<<<<< HEAD
```
Comment (OLI9292) [inline src/Components/Home/index.js]: are you seeing all this error highlighting?  maybe because of this `<<<<<<< HEAD` here

**Comment by bencbernstein** [comment]
ok, this should be fine

**Pull request closed without merging**